src/meta/processors/BaseProcessor.inl

```diff
@@ -162,6 +162,18 @@ Status BaseProcessor<RESP>::spaceExist(GraphSpaceID spaceId) {
     return Status::SpaceNotFound();
 }
 
+template<typename RESP>
+Status BaseProcessor<RESP>::userExist(UserID spaceId) {
```

Comment:
Could we unify the userExist with getUserId ?

Reply:
`userExist` is judged by UserID , But  `getUserId` is  judged by userName. And they query for different tables userIndex and userData. They are same with spaceExist and getSpaceId. I think it's best not to unify them. WDYT?

Reply:
Fair enough.